addons/web/static/src/views/list/list_renderer.js

```diff
@@ -113,6 +120,9 @@ export class ListRenderer extends Component {
 
         this.cellToFocus = null;
         this.activeRowId = null;
+        this.isEditableDebug =
+            this.props.onOpenFormView && this.env.debug && !this.props.openFormViewButton;
+        this.state = useState({ debugOpenView: false });
```

Comment:
by consistency with optional fields, I would call render() in toggleDebugOpenView and don't have a state (also `state` is very common, so this could easily be broken by a custo already defining a state)